### PR TITLE
refactor: pull out invitationQueries into separate file

### PR DESCRIPTION
**Pull request by katelynsills**
refactor: pull out invitationQueries into separate file

Part of #2860

### CODE REVIEW

**Review by erights — COMMENTED** [review]
Please let me know what you think of #2997 as an alternate way of doing this.

File: packages/zoe/src/zoeService/invitationQueries.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// @ts-check
+
+import { assert, details as X } from '@agoric/assert';
+import { E } from '@agoric/eventual-send';
+
+export const makeInvitationQueryFns = invitationIssuer => {
+  /** @type {GetInvitationDetails} */
+  const getInvitationDetails = async invitationP => {
+    const onRejected = reason => {
+      const err = assert.error(X`A Zoe invitation is required, not ${invitationP}`);
```
Comment (erights) [inline packages/zoe/src/zoeService/invitationQueries.js]: Sorry, I should have run `yarn lint-fix` before merging my suggestion into your PR.

**Review by erights — APPROVED** [review]
LGTM

File: packages/zoe/src/zoeService/invitationQueries.js (hunk 1 above)
Comment (katelynsills) [reply packages/zoe/src/zoeService/invitationQueries.js]: No worries, thanks for your PR!

**Pull request merged**